src/pages/fieldManagement/settings/saveField/OptionEditor.jsx

```diff
@@ -112,7 +117,25 @@ export default function OptionEditor({
         </Button>
       </DialogContent>
       <DialogActions style={{ padding: '0px 24px 24px 24px' }}>
-        <Button display="primary" onClick={onSubmit}>
+        <Button
+          disabled={
+            displayedOptions.filter(
+              option => !option?.value || !option?.label,
+            ).length > 0
+          }
+          showTooltip
+          tooltipText={
+            displayedOptions.filter(
+              option => !option?.value || !option?.label,
+            ).length > 0
+              ? intl.formatMessage({
+                  id: 'UNFINISHED_OPTIONS',
+                })
+              : ''
+          }
```

Comment:
When the options are valid, there is no need for a Tool tip.

Reply:
Changed in https://github.com/WildMeOrg/codex-frontend/pull/412/commits/b2c0f77857f9e4d41298476ae26b4473b092be20